tests/integration/original_metadata/ERP001178.csv.md5

```diff
@@ -1 +1 @@
-102059fc508c6c3f950ee62dbb3cbc72  ERP001178.csv
+7b26ba440c42b3f8b9b41c378b5a36be  ERP001178.csv
```

Comment:
how did the md5 change if the csv fixture hasn't changed?

Reply:
there is something funky going one

Reply:
The column order was causing issues.